### PR TITLE
Add Intellisense in proxy.config

### PR DESCRIPTION
**Pull request by nicogis**
Add Intellisense VS in proxy.config

### CODE REVIEW

**Comment by bsvensson** [comment]
Thanks @nicogis for helping improve the resource-proxy!

**Comment by bsvensson** [comment]
FYI: it looks like after this commit, users of the DotNet proxy, need to update their proxy.config as in the commit https://github.com/Esri/resource-proxy/pull/486/commits/3f000007f69cadd3476238b2894e2714429744f4#diff-619e6e6c6786d5cebe866bfb1ab4492f

If you update your resource-proxy for DotNet, but keep your old proxy.config as is, your proxy will probably break saying `[InvalidOperationException: <ProxyConfig xmlns=''> was not expected.]`

Basically, the DotNet proxy is now "stricter", and you just have to change from
`<ProxyConfig allowedReferers="*" mustMatch="false">`
in your proxy.config to
`<ProxyConfig allowedReferers="*" mustMatch="false" xmlns="proxy.xsd">`
